src/lib/dnssd/Resolver_ImplMinimalMdns.cpp

```diff
@@ -363,14 +363,15 @@ void MinMdnsResolver::AdvancePendingResolverStates()
     {
         if (!resolver->IsActive())
         {
+            ChipLogError(Discovery, "resolver inactive, continue to next");
```

Comment:
It's a normal situation in the case of parallel queries, not an error, right?

Reply:
removed.